modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/util/comparator/WorkflowDefinitionNameComparator.java

```diff
@@ -20,12 +22,14 @@ public class WorkflowDefinitionNameComparator
 
 	public static final String[] ORDER_BY_FIELDS = {"name"};
 
-	public WorkflowDefinitionNameComparator() {
-		this(false);
-	}
+	public static WorkflowDefinitionNameComparator getInstance(
```

Comment:
@victorhcunha @titokenzo The change to this file seems to be out of scope of the PR. And since it's internal with no usage, when you work on it, you don't need to deprecate it; you can just remove it.

Reply:
This change is part of the PR but should be in a different commit (I resent it). By default, we're just deprecating the unused classes. After your review, I realized that some were missing this, so I added it.
But about this one, since it's not really exported, should I remove it completely? If so, I can resend it with the change.